File: app/src/main/java/com/developer/valyutaapp/ui/main/MainViewInterface.java
package com.developer.valyutaapp.ui.main;

import com.developer.valyutaapp.model.Valute;

import java.util.List;

public interface MainViewInterface {
    void showToast(String s);
    void showProgressBar();
    void hideProgressBar();
    void displayValutes(List<Valute> valutes);
    void displayError(String s);
}

File: app/src/main/java/com/developer/valyutaapp/ui/main/MainViewInterface.kt
package com.developer.valyutaapp.ui.main

import com.developer.valyutaapp.domain.entities.Valute

interface MainViewInterface {
    fun showToast(s: String?)
    fun showProgressBar()
    fun hideProgressBar()
    fun displayValutes(valutes: List<Valute>?)
    fun displayError(s: String?)
}